crates/aptos-faucet/metrics-server/README.md

```diff
@@ -0,0 +1 @@
+Note: Much of this is yanked from https://github.com/aptos-labs/aptos-core/tree/main/crates/inspection-service. Unfortunately adding a dependency on that meant bringing in the whole Aptos codebase, including stuff that would require me to enable tokio unstable (not a huge deal, but just an indicator of how much extra code I'd be bringing in). So this is a stripped down, more tokio native version of the inspection service.
```

Comment:
Probably can remove this, or use the inspection service

Reply:
I don't want to make more major changes in this PR, I'm trying to make it as much of a drop in as possible. We can cut over in a later PR. Though tbh this metrics server I have is better for how we use it here, the other is needlessly complex / heavy.

Reply:
Also the inspection service is coupled pretty heavily to a node, it assumes you'll have a NodeConfig and whatnot.